cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -260,12 +287,16 @@ private boolean removeFollowerThrottledReplicasFromTopic(Properties config, Stri
   }
 
   private void removeThrottledReplicasFromTopic(String topic, Set<String> replicas) {
-    Properties config = _kafkaZkClient.getEntityConfigs(ConfigType.Topic(), topic);
-    boolean removedLeaderThrottle = removeLeaderThrottledReplicasFromTopic(config, topic, replicas);
-    boolean removedFollowerThrottle = removeFollowerThrottledReplicasFromTopic(config, topic, replicas);
+    Properties topicConfigs = _kafkaZkClient.getEntityConfigs(ConfigType.Topic(), topic);
+    if (topicConfigs == null || topicConfigs.isEmpty()) {
+      LOG.warn("Skip removing throttled replicas {} from topic {} since no configs can be read", String.join(",", replicas), topic);
```

Comment:
When would this be expected to happen? 
Here are some scenarios that I can think of:
  (1) a topic is deleted and then shortly after recreated with the same name with no config,
  (2) someone externally overrides the configs,
  (3) a programming error

If it happens due to (1) or (2), are users expected to do anything if they see this warning log? Does the way we handle it suffice to provide the right behavior?
Assuming that we have proper testing so that (3) is unlikely, should we make this a `debug` level log so that we don't cause extra logs that are not actionable?

Reply:
OK, I agree that we can change it to `debug` level